srcpkgs/kindd/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'kindd'
+pkgname=kindd
+version=2.1.0
+revision=1
+wrksrc="kindd-${version}"
```

Comment:
Just remove this line, this is the default value.

Reply:
Done!